examples/aws-databricks-modular-privatelink/databricks_account_tf_pipeline/modules/account_log_delivery/log_delivery.tf

```diff
@@ -0,0 +1,107 @@
+terraform {
```

Comment:
do we still suggest customer to configure log delivery? system tables should replace this now as parsing the audit logs is quite time-consuming